blue_hardware_drivers/include/blue_hardware_drivers/comms_defs.h

```diff
@@ -108,6 +109,7 @@ constexpr comm_addr_t COMM_REG_VOL_PHASE_C_PWM = 0x2005;
 constexpr comm_addr_t COMM_REG_VOL_SETPOINT_T = 0x2006;
 constexpr comm_addr_t COMM_REG_VOL_SETPOINT_V = 0x2007;
 constexpr comm_addr_t COMM_REG_VOL_SETPOINT_P = 0x2008;
+constexpr comm_addr_t COMM_REG_VOL_SETPOINT_FF = 0x2009;
```

Comment:
Was this a mistake? It has the same address as `COMM_REG_VOL_PWM_DRIVE`, and I don't see it being used anywhere